docs-new/docs/graphql/core/auth/authorization/basics.mdx

```diff
@@ -1,63 +1,63 @@
-<div class="meta" description="Hasura access control basics"
-keywords="hasura, docs, authorization, access control">
-
-</div>
+---
+description: Hasura access control basics
+keywords:
+  - hasura
+  - docs
+  - authorization
+  - access control
+sidebar_position: 2
+---
+
+import Thumbnail from "@site/src/components/Thumbnail/Thumbnail";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
 # Access control basics
 
-<div class="contents" backlinks="none" depth="2" local="">
-
-Table of contents
-
-</div>
-
 ## Introduction
 
 To understand the basics of access control in Hasura, let's take a look
 at this analogy to a SQL query:
 
-<div class="thumbnail" width="700px"
-alt="Understanding access control in Hasura">
-
-/img/graphql/core/auth/permissions-rule-analogy.png
-
-</div>
+<Thumbnail
+  src="/img/graphql/core/auth/permissions-rule-analogy.png"
+  alt="Understanding access control in Hasura"
+  width="700px"
+/>
 
 This query returns the right set of results by defining the requirements
 for columns and rows in a given table. Hasura's rule-based access
 control works similarly - you define the following permissions for a
-combination of **role**, **table** and **action** (*insert, update,
-select and delete*):
+combination of **role**, **table** and **action** (_insert, update,
+select and delete_):
 
 **Row-level permissions**
 
 Limit access to a subset of the rows in the table based on this
 permission. Row-level permissions are essentially boolean expressions
 that, when evaluated against any row, determine access to it. These
 permissions are constructed from the values in columns,
-`session variables <roles_variables>` and static values to build this
+[session variables](/graphql/core/auth/authorization/roles-variables.mdx/) and static values to build this
```

Comment:
pls avoid `/` suffix for the file paths.

Here and all other occurrences in auth/authorization directory.